### DISCUSSION TITLE
Make OpenAm container with persistent installation

### DISCUSSION THREAD

**Discussion by vugrinov** [OpenIdentityPlatform/OpenAM#431] [Q&A] [answered]
Hi, successfully  runing an OpenAM container but affter i restart the machine all the installation data is gone ...
I am attaching 2 volumes but still ... the installation data does not persist.
`docker run \
--restart unless-stopped \
-h openam-dev.xxxxxxxxx \
--volume=/dockerfs/projects/openam_dev:/TAR/openam_xxxx:rw \
--volume=/dockerfs/projects/openam_conf2:/usr/openam/config:rw \
-d -p xx.xx.xx.xx:8989:8080 \
--name openam-dev-test openidentityplatform/openam:14.6.4`

**Comment by hamzahassan51** [comment, marked as answer]
OpenAM configuration directory by default is in /usr/openam/config folder, you can mount it as a volume for persistence. You can also change the directory via $OPENAM_DATA_DIR env variable.
